projects/js-packages/charts/src/components/pie-chart/pie-chart.tsx

```diff
@@ -0,0 +1,130 @@
+import { Group } from '@visx/group';
+import { Pie } from '@visx/shape';
+import { SVGProps } from 'react';
+import useChartMouseHandler from '../../hooks/use-chart-mouse-handler';
+import { useChartTheme, defaultTheme } from '../../providers/theme';
+import { Tooltip } from '../tooltip';
+import type { DataPoint } from '../shared/types';
+
+// TODO: add animation
+
+type PieChartProps = {
```

Comment:
We might want to centralize the typings. Another thing is, would it make sense to use interfaces with inheritance to reduce dup params of all the charts?

Reply:
That's a good idea. I wanted to revisit props once we make charts responsive and remove width and height but I can add the interface now.